Hakawai/Core/HKWTextView.h

```diff
@@ -43,8 +43,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (BOOL) enableExperimentalDeadLockFix;
 + (BOOL) enableMentionSelectFix;
++ (BOOL) enableSimpleRefactor;
```

Comment:
not an issue. 
naming wise, is it better to name it cursor movement refactor?

Reply:
yes will fix in umpcoming rb